compiler/crates/relay-typegen/src/visit.rs

```diff
@@ -473,7 +477,13 @@ fn get_resolver_arguments(
     encountered_enums: &mut EncounteredEnums,
     custom_scalars: &mut std::collections::HashSet<(StringKey, PathBuf)>,
     schema_field: &Field,
+    context_import: Option<LiveResolverContextType>,
 ) -> Vec<KeyValuePairProp> {
+    let void_type = match typegen_context.project_config.typegen_config.language {
+        TypegenLanguage::Flow | TypegenLanguage::JavaScript => AST::RawType(intern!("void")),
+        TypegenLanguage::TypeScript => AST::RawType(intern!("undefined")),
+    };
+
```

Comment:
In javascript, we probably don't want to show any types at all? Or we could have TS docblock types, but in that case perhaps it would make more sense to align JS with TS?

Reply:
Good catch. This might be caught later in the writer for JS also just by never writing any types but I'm not sure about that.

Reply:
Would be good to add a test which confirms what happens here. Could be an integration test or a typegen test